docker/Dockerfile

```diff
@@ -67,19 +70,33 @@ ARG PHENIX_TAG
 RUN COMMIT=${PHENIX_COMMIT} TAG=${PHENIX_TAG} make bin/phenix
 RUN make -C src/go phenix-tunneler
 
-ARG APPS_REPO=sandialabs/sceptre-phenix-apps
+ARG APPS_REPO=github.com/sandialabs/sceptre-phenix-apps
 ARG APPS_BRANCH=main
 
-RUN git clone --branch ${APPS_BRANCH} https://github.com/${APPS_REPO}.git /phenix-apps
+RUN git clone --branch ${APPS_BRANCH} https://${APPS_REPO}.git /phenix-apps
 
 WORKDIR /phenix-apps/src/go
 
 RUN CGO_ENABLED=0 GOOS=linux go install -trimpath ./...
 
 
+# ** Build the phenix image **
 FROM ubuntu:22.04
 
-ENV TZ=Etc/UTC
+# General image metadata
+# Reference: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+ARG PHENIX_COMMIT
+LABEL org.opencontainers.image.authors="Sandia National Laboratories <emulytics@sandia.gov>" \
```

Comment:
Since I'm one of the main authors, but not part of the emulytics@sandia.gov mailing list, can I get my name and email address added here as well?

Reply:
Definitely. I added your name with the activeshadow email from your git commits. If there's another you'd prefer, let me know.